mllib/src/test/scala/org/apache/spark/mllib/linalg/MatricesSuite.scala

```diff
@@ -46,6 +46,26 @@ class MatricesSuite extends SparkFunSuite {
     }
   }
 
+  test("Test Breeze Conversion Bug - SPARK-20687") {
```

Comment:
specific name: Test FromBreeze when Breeze.CSCMatrix.rowIndices has trailing zeros.

And move the test after another unit test "fromBreeze with sparse matrix"